MockKit/MockPumpManager.swift

```diff
@@ -358,14 +374,22 @@ public final class MockPumpManager: TestingPumpManager {
     public func enactTempBasal(unitsPerHour: Double, for duration: TimeInterval, completion: @escaping (PumpManagerResult<DoseEntry>) -> Void) {
         logDeviceComms(.send, message: "Temp Basal \(unitsPerHour) U/hr Duration:\(duration.hours)")
 
-        if state.tempBasalEnactmentShouldError {
+        if state.tempBasalEnactmentShouldError || state.pumpBatteryChargeRemaining == 0 {
```

Comment:
Should there be a check for `reservoirUnitsRemaining`?

Reply:
oh.... yes. will add